experimenter/experimenter/nimbus_ui_new/views.py

```diff
@@ -264,3 +271,93 @@ class UnsubscribeView(
 ):
     form_class = UnsubscribeForm
     template_name = "nimbus_experiments/subscribers_list.html"
+
+class TimelineAndControlsMixin:
```

Comment:
So I think a different way to do this is to move all this logic directly into the summary template, and then use hx-select and hx-target to just update the controls/timeline, rather than doing logic in the views here.